lib/core/rdp.js

```diff
@@ -42,7 +42,18 @@ Rdp.registerGlobalActor = function(client, options) {
     return;
   }
 
-  client.listTabs(response => {
+  let listTabs = {
+    to: "root",
+    type: "listTabs"
+  };
+
+  // NOTE: the following option asks firebug.sdk to mark custom actors registering RDP packets
+  // as rdpInspectorInternals (which helps to filter out them from the packet list)
+  if (options.rdpInspectorInternals) {
```

Comment:
@janodvarko I'm not sure this is the best solution (because it is specific to the rdp-inspector and it sounds a bit out of scope inside the firebug.sdk).

I can turn this into a more generic ```options.customAttributes``` (which, if specified, will be merged into the json RDP packet), how it sounds to you?